### ISSUE TITLE
make it so it can steal intellij accounts

### ISSUE THREAD

**Issue by master7720** [Sxmurai/aesthetirat#4]

**Comment by Notme11** [comment]
💀💀💀💀💀💀💀💀💀

**Comment by HausemasterIssue** [comment]
:skull: :skull: :skull: :skull: :skull: :skull: :skull:

**Comment by Sxmurai** [comment]
Do not worry I will add shit to this rat, I'll even make an example on how
to use it in a classic OyVey base.

**Comment by master7720** [comment]
bruh 😭

**Comment by master7720** [comment]
> 💀 💀 💀 💀 💀 💀 💀

is this going way too far :neckbeard:

**Comment by HausemasterIssue** [comment]
nah, we gotta get those classic ratted oyvey renames that these guys are keeping private :skull: :skull: :skull: 💀 :skull: :skull: :skull: :skull:
